simulator/src/config_parser.py

```diff
@@ -63,7 +63,7 @@ def parse_config(config_file_path):
         fp.close()
         sys.stdout.write(e)
         sys.stdout.write("Could not read data, please check if all fields are filled")
-        return
+        raise ValidationError(str(e))

```

Comment:
sys.stdout.write(e) can be removed, all the messages can be combined and raised as ValidationError.

eg. ValidationError('error reading config file - ' + str(e))